tests/integration_test.rs

```diff
@@ -202,7 +210,9 @@ mod tests {
             + &jwt_payload_bad_data_plane_base64.clone()
             + "."
             + &jwt_invalid_signature_base_64.clone();
-        let mut client = SimpleCacheClient::new(bad_data_plane_jwt, 5).await.unwrap();
+        let mut client = SimpleCacheClient::new(bad_data_plane_jwt, NonZeroU64::new(5).unwrap())
```

Comment:
so there is no way user can just pass `5` any more they have to explicitly cast to `NonZeroU64`? I really like the compile time check just kinda anoying user needs to pass this everytime now makes our code little more wordy.

Reply:
Yes, now users need to pass `NonZeroU64::new(5).unwrap()` instead of just `5`.
Although passing 0 to `NonZeroU64::new(0).unwrap()` does not give a compile time error, users should be able to understand what get returned if they pass 0 based on [this](https://doc.rust-lang.org/stable/std/num/struct.NonZeroU64.html#method.new).